cookbooks/bcpc-hadoop/templates/default/flume_flume-conf.erb

```diff
@@ -1,5 +1,6 @@
 <%= @agent_name %>.channels = memoryChannel
 <%= @agent_name %>.channels.memoryChannel.type = memory
+<%= @agent_name %>.channels.memoryChannel.capacity = 10000
```

Comment:
@kiiranh the memory channel need to have enough capacity to offset the entries which will be added to it in the time delay for the HDFS sink to write data to HDFS. The default value of channel capacity is 100 which is too low when HDFS writes takes some delay but should not take more than few seconds in worst case scenario. If we assume that the worst case scenario is 20 secs and we can expect that the memory channel will have to hold around 2000 entries max. Hence the decision to go with a number of 10000 which will be able to handle a much bigger delay in HDFS write completion.